src/components/JsonSchema/JsonSchemaQA.tsx

```diff
@@ -34,18 +34,18 @@ function extractTextFromBlocks(blocks: BlocksProps[]) {
 }
 
 interface QAJsonSchemaProps {
-  data: FaqMeta[];
+  data: FaqProps[];
 }
 
 export const QAJsonSchema = ({ data }: QAJsonSchemaProps) => {
   const schema = useMemo(() => {
     const entities = data?.map((el, index) => {
-      const text = extractTextFromBlocks(
-        el.attributes.Answer as unknown as BlocksProps[],
-      );
+      const text = el.Answer;
+      // used for data from strapi -->
```

Comment:
do we nee to keep it?

Reply:
If you mean the comment, it´s removed.

Generally on the JsonSchemaQA.tsx -> It´s meant to be a feature, not a bug.
There is a JSON schema for FAQs which is meant to improve search results. Our FAQ-component also makes use of the content and converts it into such a schema.

1.) As an example on our current FAQ on swap pages:

`{"@context":"https://schema.org","@type":"FAQPage","mainEntity":[{"@type":"Question","name":"Some Question here 1","acceptedAnswer":{"@type":"Answer","text":"Some Answer here 1"}},{"@type":"Question","name":"Some Question here 2","acceptedAnswer":{"@type":"Answer","text":"Some Answer here 2"}}]}`

2.) You can paste it here to double test and validate the output:
https://developers.google.com/search/docs/appearance/structured-data/faqpage?hl=en